### ISSUE TITLE
Crash when resuming the app

### ISSUE THREAD

**Issue by okainov** [cgeo/cgeo#7378]
##### Detailed steps causing the problem:
- Run c:geo
- Do some cache stuff
- "minimize" it, do other stuff in other apps (navigation in my case)
- Try to switch back to already running c:geo

##### Actual behavior after performing these steps:

Crash, "app was stopped", "app still stopping"

##### Expected behavior after performing these steps:

C:geo resumes normally

##### Is the problem reproducible for you?
Yes

##### System information:
```
Attach system information here if available (see c:geo menu -> About c:geo -> Swipe right to System)
Keep the apostrophe at beginning and end to have it properly formatted
```

**Comment by Lineflyer** [comment]
Might be related or duplicate of #7300

**Comment by rsudev** [comment]
Can you define more precisely which activity was open when the app was sent to the background? Can you give a step-by-step description? What version of c:geo did you use?

**Comment by okainov** [comment]
@rsudev map activity most of the time. I think once it was cache description but then I had no crash (as @Lineflyer wrote, can be really the same as #7300)
No I cannot because sometimes (maybe when there was not enough time and app was still "active") it works, but often I'm getting crash.

Master from couple.of days ago + su.

**Comment by moving-bits** [comment]
Is this one fixed now by either #7428 or #7457?

**Comment by Lineflyer** [comment]
I assume so. @okainov feel free to reopen if you encounter this again.
